src/webview/index.ts

```diff
@@ -36,6 +36,7 @@ window.addEventListener(
     () => {
         addListeners();
         TranslationUpdater();
+        vscode.postMessage({ type: "update-config" });
```

Comment:
This fix is not fixing anything. Lets see the events on main process

https://github.com/ArthurLobopro/easy-codesnap/blob/954bdcbe55cdbb913d045b1aa56498e950f329f5/src/commands/snap/SnapActions.ts#L44-L53

The event "ready" already loads the config that "update-config" do, but the "ready" event load another things togheter. So call "update-config" is not a fix because is redundant

Reply:
It is redundant unless user hasn't selected anything yet. Capturing without any selection triggers an error on the client side because there are no config available.

That said, I missed the `ready` action itself. Maybe updating config inside `ready` might suffice?

Reply:
I don't like the option to open the screen without something selected. I am thinking of block this behavior, it is actually the target bug in my opinion.

Reply:
I created a branch fixing that: https://github.com/ArthurLobopro/easy-codesnap/tree/bug-fixes. You can submit a PR there fixing the imports. After we can add more options to save scale and release a new version

Reply:
Maybe it should be something like blocking capture action when nothing is selected? Ok, I'll revert this fix.

Reply:
> I created a branch fixing that: https://github.com/ArthurLobopro/easy-codesnap/tree/bug-fixes. You can submit a PR there fixing the imports. After we can add more options to save scale and release a new version

Opened: #96.